english/explore-outside/investigate-marshmallow-structure/greet-green/greet-green.md

```diff
@@ -1,5 +1,8 @@
 You find yourself a way to the tower to greet the friendly looking green people. But their
-reaction is not quite what you expected. Upon seeing you, they cease their dance and douse 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.